whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -463,4 +465,38 @@ private static VCard createVCard(final RpslObject rpslObject) {
 
         return builder.build();
     }
+
+    private static void handleLanguageAttribute(RpslObject rpslObject, RdapObject rdapObject) {
```

Comment:
Please use final in method parameters and variables, so they can't be re-assigned.